storages/python/python-storage-plugins/redis.py

```diff
@@ -17,7 +17,10 @@ def open(self, location: str, options=None):
             - `port`: Port to connect to (default: 6379).
             - `username`: Redis user name.
             - `password`: Redis password.
-            - `ssl`: Whether to ssl-encrypt the connection (default: false).
+            - `socket_keepalive`: Whether to enable socket keepalive option.
+              Seems to protect against hanging.  Default: true
+            - `socket_timeout`: Timeout after given number of seconds.
+            - `ssl`: Whether to ssl-encrypt the connection.
```

Comment:
Changing the ssl configuration was not mentioned in the commit messages and is not described elsewhere. Was this necessary for fixing the "redis hanging" problem?

Previously, it was clear that the `ssl` setting was `false` by default. Now, the default behavior is undefined as far as I can tell. Is this the intention?

Reply:
It was a part of my testing around for a solution for the hanging problem - but has nothing to do with the hanging problem.

However, I kept ssl unset, since I think it is a bad habit to turn off security by default. Now we leave it to redis to select the default value. I believe they have a better feeling for a sensible default value than I have.

Reply:
OK, so the security change is intentional. For traceability it would have been better to do this in a dedicated PR, but let's keep it in this one anyway.